protocol/src/main/java/org/fidoalliance/fdo/protocol/api/OwnerVoucher.java

```diff
@@ -4,24 +4,72 @@
 package org.fidoalliance.fdo.protocol.api;
 
 import jakarta.servlet.http.HttpServletResponse;
+import java.io.IOException;
 import java.util.Date;
+import java.util.List;
 import java.util.logging.Logger;
 
+import javax.persistence.TypedQuery;
+import javax.persistence.criteria.CriteriaBuilder;
+import javax.persistence.criteria.CriteriaQuery;
+import javax.persistence.criteria.Root;
 import org.fidoalliance.fdo.protocol.HttpUtils;
 import org.fidoalliance.fdo.protocol.LoggerService;
 import org.fidoalliance.fdo.protocol.Mapper;
+import org.fidoalliance.fdo.protocol.StandardTo0Client;
 import org.fidoalliance.fdo.protocol.VoucherUtils;
+import org.fidoalliance.fdo.protocol.db.HibernateUtil;
+import org.fidoalliance.fdo.protocol.db.OnboardConfigSupplier;
 import org.fidoalliance.fdo.protocol.entity.CertificateData;
+import org.fidoalliance.fdo.protocol.entity.OnboardingConfig;
 import org.fidoalliance.fdo.protocol.entity.OnboardingVoucher;
 import org.fidoalliance.fdo.protocol.message.OwnershipVoucher;
 import org.fidoalliance.fdo.protocol.message.OwnershipVoucherHeader;
+import org.fidoalliance.fdo.protocol.message.To0OwnerSign;
+import org.fidoalliance.fdo.protocol.message.To0d;
+import org.fidoalliance.fdo.protocol.message.To2AddressEntries;
+import org.hibernate.Session;
 
 /**
  * Maintains Ownership Vouchers for the owner server.
  */
 public class OwnerVoucher extends RestApi {
 
-  protected static LoggerService logger = new LoggerService(OwnerVoucher.class);
+  protected static final LoggerService logger = new LoggerService(OwnerVoucher.class);
+
+  private void listVouchers() {
+
+    final Session session = HibernateUtil.getSessionFactory().openSession();
+    try {
+
+      CriteriaBuilder cb = session.getCriteriaBuilder();
+      CriteriaQuery<OnboardingVoucher> cq = cb.createQuery(OnboardingVoucher.class);
+      Root<OnboardingVoucher> rootEntry = cq.from(OnboardingVoucher.class);
+      CriteriaQuery<OnboardingVoucher> all = cq.select(rootEntry);
+
+      TypedQuery<OnboardingVoucher> allQuery = session.createQuery(all);
+      List<OnboardingVoucher> list = allQuery.getResultList();
+      OnboardingConfig onboardConfig = new OnboardConfigSupplier().get();
```

Comment:
onboardConfig - Variable was never read after being assigned.

Reply:
fixed in commit